cst438_gradebook/src/main/java/com/cst438/controllers/GradeBookController.java

```diff
@@ -30,6 +30,7 @@
 import com.cst438.domain.CourseRepository;
 import com.cst438.domain.Enrollment;
 import com.cst438.domain.GradebookDTO;
+import com.cst438.services.AssignmentService;
```

Comment:
Not sure this addition is needed? I think we just creating abstractions for each-other's services. 
So in my repo I have `GradebookServiceREST` and `GradebookServiceMQ`, and in your repo you have `RegistrationService` - REST and MQ.

Reply:
I think I was testing something and did some imports